include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort_one_wg.h

```diff
@@ -88,18 +88,16 @@ struct __subgroup_radix_sort
         }
     };
 
-    template <typename _KeyT, typename _Wi, typename _Src, typename _Keys>
+    template <typename _Wi, typename _Src, typename _Keys>
     static void
-    __block_load(const _Wi __wi, const _Src& __src, _Keys& __keys, const uint32_t __n, const _KeyT& __default_key)
+    __block_load(const _Wi __wi, const _Src& __src, _Keys& __keys, const uint32_t __n)
```

Comment:
Could you explain what we are doing here? Is that your changes or incorrect merge?

Reply:
It is the change. We don't need  default key anymore.
If you have questions I can explain it offline.